### DISCUSSION TITLE
Use sortField with html attribute

### DISCUSSION THREAD

**Discussion by JeremieBergeron** [orchidjs/tom-select#410] [Q&A]
Is it possible to use the property [sortField](https://tom-select.js.org/docs/) directly with html attribute?

I am trying to wrote `data-sort-field="[{field:'$order'},{field:'$score'}]"`, but it does not seems to be applied.

**Comment by JeremieBergeron** [comment]
Since it work without any problem when I initialize my tom select with javascript: `config.sortField = [{ field: '$order' }, { field: '$score' }]`

It is probably a bug or simply a feature tom select does not provide.

I opened a issue: https://github.com/orchidjs/tom-select/issues/415

**Comment by samsong** [comment]
Hi Team
Seems there is a bug with sorting.
I have 2 input type "disabled"
when i render the page. the sorting method does NOT work.

the disabled inputs inside an optgroup are sorted to the top of the optgroup.
the active inputs are shown below.

this is not what we want. we want tomselect to respect & follow the same input order as we introduced by the html.
why is this so difficult for disabled inputs?
